src/components/propositionalLogic/examplePage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import styles from "./propositionalLogic.module.scss";
+
+class ExamplesPropositionalLogic extends React.Component {
+	public render() {
+		return <div className={styles.site}>
+			<div className={styles.proofEditor}>
+				<p><button className={styles.buttons}
+					onClick={this.showElement}>
+					Beispiel 1
+						</button></p>
+				<table className={styles.toggleTable} id="toggleTable">
+					<tr>
+						<th>Eingabe</th>
+						<th>Erklärung (mit der Maus drüberfahren zum anzeigen)</th>
+					</tr>
+					<tr>
+						<th>Wir zeigen: a -> (b -> c) -> (a und b) -> c.</th>
+						<th>Dies soll bewiesen werden. </th>
+					</tr>
+					<tr>
+						<td></td>
```

Comment:
It looks good, but it could potentially be solved differently. This way it is an irregularity which can make it harder to process this structur. But I guess the way it is done here is okayish in this case. Not an issue I would block an merge for.